RSFormView/Classes/Views/TextField/TextFieldView.swift

```diff
@@ -166,7 +166,15 @@ public class TextFieldView: UIView {
   func populateTextView(withData data: FormField) {
     updatePlaceHolder(withText: data.placeholder)
     textField.clearButtonMode = (data.fieldType == .date || data.fieldType == .picker) ? .never : .whileEditing
-    textField.isSecureTextEntry = data.fieldType == .password
+    
+    //This doesn't make any sense, but for a project this line:
+    //textField.isSecureTextEntry = data.fieldType == .password, was masking every field :shrug:
```

Comment:
Cell reuse issue maybe?? 🤔

Reply:
no, this method is called on cellForRow... Also if that was the case it should still be broken with this "fix"... this is some twilight zone shit